### PR TITLE
P20-990: Fix Policy Compliance UI tests

### PR DESCRIPTION
**Pull request by artem-vavilov**
## Links:
- JIRA task: [P20-990](https://codedotorg.atlassian.net/browse/P20-990)
- Related PR: https://github.com/code-dot-org/code-dot-org/pull/59637

### CODE REVIEW

File: dashboard/test/ui/features/platform/policy_compliance.feature [hunk 1]
```diff
@@ -150,13 +155,16 @@ Feature: Policy Compliance and Parental Permission
...
+    Given I am on "http://studio.code.org"
+    # CPA new user lockout phase starts 1 month before CPA exception
+    Given CPA new user lockout phase starts at "2024-04-26T00:00:00MDT"
+
+    When I create a young student in Colorado who has never signed in named "Sally Student" before CPA exception and go home
+    Then I am on "http://studio.code.org/home"
 
   Scenario: Teacher should be able to connect a third-party account even without a state specified
-    Given I create a teacher who has never signed in named "Amstrad Teacher" after CPA exception and go home
+    Given I create a teacher who has never signed in named "AMDTrad Teacher" after CPA exception and go home
```
Comment (wilkie) [inline dashboard/test/ui/features/platform/policy_compliance.feature]: Haha. I thought maybe you were rebelling against my use of 80s computer models as my user naming scheme in this file until I realized what happened here. 😂

**Review by wilkie — APPROVED** [review]
Your mst -> mdt substitution hit a username 😂 but otherwise this all looks great and I'm very happy to see the DCDO mocking being used very very effectively here.

File: dashboard/test/ui/features/platform/policy_compliance.feature (hunk 1 above)
Comment (artem-vavilov) [reply dashboard/test/ui/features/platform/policy_compliance.feature]: Sorry, I'll fix this!

**Review by wilkie — APPROVED** [review]

**Review by wilkie — CHANGES_REQUESTED** [review]
Just to remember to remove the console.logs

**Review by wilkie — APPROVED** [review]

**Pull request merged**